web/src/locales/zh-TW.json

```
@@ -114,9 +114,8 @@
   "estimation-card": {
     "link": "了解我們的估計模型",
     "estimated_time_slicer_average": {
-      "title": "數據目前為估計值",
-      "pill": "不可用",
-      "body": "數據提供者目前沒有報告這一小時的數據。 顯示的值是根據歷史測量值估計的，並將在可用時替換為綜合數據。 請注意，估計值可能並不總是準確的，因為它們沒有考慮到特定的即時天氣事件。"
+      "title": "",
+      "body": ""
```

@madsnedergaard , I have provided a new translation for the title and body. Please review the changes and let me know if there are any further adjustments needed. :)
```suggestion
      "title": "數據為初步的值",
      "body": "資料提供者未提供此小時的數據。目前顯示的數值是基於歷史測量值估計的，一旦實際數據可用將立即更新。\n\n請注意，估計值可能未考慮到即時的天氣事件。"

---

This is great, thanks! 🙏